community/portworx/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 name: portworx
-version: 1.0.18
+version: 1.0.19
 description: A Helm chart for installing Portworx on Kubernetes.
 tillerVersion: ">=2.7.2"
 kubeVersion: ">=1.10.0"
-appVersion: "2.5.1"
```

Comment:
Change this to match the imageVesion i.e the relase version of Porx we are installing.

Reply:
updated